### ISSUE TITLE
Take another look at the shortcut overlay

### ISSUE THREAD

**Issue by JulianKniephoff** [elan-ev/opencast-studio#1098]
@lkiesow wrote in https://github.com/elan-ev/opencast-studio/pull/1069#issuecomment-1691152010:

> ![Screenshot from 2023-08-24 09-08-32](https://github.com/elan-ev/opencast-studio/assets/1008395/f349647f-f5a5-426a-a973-dc85f1a64f60)
>
> I like the idea of highlighting shortcuts, even though I'm pressing `alt` all the time to use accessibility tools. That may be why I realized that there seem to be a number of things missing. It seems odd that e.g. “Shortcuts” has a shortcut, but the buttons right next to that do not.
>
> This also goes for other places in the editor like e.g. the settings (gear) icons.

@LukasKalbertodt responded in https://github.com/elan-ev/opencast-studio/pull/1069#issuecomment-1691246311:

> Fair. I'm happy to add more shortcuts in the future. I'm not sure all places you mention really deserve their own shortcut, but that's a discussion we can have after this PR is merged.

@lkiesow added in https://github.com/elan-ev/opencast-studio/pull/1069#issuecomment-1693065913:

> The problem is mostly that you introduced the shortcut overlay, which I think looks very odd in its current form. At least on the main menu.

I'm not sure whether this issue is supposed to be mainly about the missing/additional shortcuts, or whether @lkiesow's last comment in particular wants to express some more concerns. Can you elaborate?

**Comment by lkiesow** [comment]
To give you a specific example, whenever you press `<alt>` (I do that all the time to access some accessibility options) you now get an overlay showing you potential shortcuts. That's new. Before you just had some shortcuts for functionality that people deemed important. This leads to the user interface looking like this (if you look, for example, to the main menu):

![Screenshot from 2023-09-14 12-21-54](https://github.com/elan-ev/opencast-studio/assets/1008395/2fb3081d-8f2a-4a0a-bb46-6a8e9a413ddc)

So, I get an overlay over one of four icons in the main menu. That looks pretty weird.

What's the solution to this? You could remove the overlay again. Or you could add shortcuts to all menu items. It's just that in it's current form, it's not great.

**Comment by LukasKalbertodt** [comment]
Closing as part of issue cleanup: there has been no further discussion and the shortcut overview has been out there for almost 2 years, so users are now used to it. Removing it is not really an option I think. (Editor has the same). I do understand it's kind of meh that only one of four header buttons has a shortcut, but that's rather minor. If someone really wants to "fix" this still, I think the only option is to add shortcuts for the others. But that's a different issue/PR.
